gamestonk_terminal/forex/fx_view.py

```diff
@@ -0,0 +1,544 @@
+    parameters["count"] = ns_parser.count
+
+    try:
+        request = orders.OrderList(accountID, parameters)
+        response = client.request(request)
+        df = pd.DataFrame.from_dict(response["orders"])
+        df = df[["id", "instrument", "units", "price", "state", "type"]]
+        print(df)
+    except V20Error as e:
+        print(e)
+
+
+def get_order_book(instrument):
+    parameters = {"bucketWidth": "1"}
+    try:
+        request = instruments.InstrumentsOrderBook(
+            instrument=instrument, params=parameters
+        )
+        response = client.request(request)
+        df = pd.DataFrame.from_dict(response["orderBook"]["buckets"])
+        pd.set_option("display.max_rows", None)
+        df = df.take(range(527, 727, 1))
+        book_plot(df, instrument, "Order Book")
+    except V20Error as e:
+        print(e)
+
+
+def get_position_book(instrument):
+    try:
+        request = instruments.InstrumentsPositionBook(instrument=instrument)
+        response = client.request(request)
+        df = pd.DataFrame.from_dict(response["positionBook"]["buckets"])
+        pd.set_option("display.max_rows", None)
+        df = df.take(range(219, 415, 1))
+        book_plot(df, instrument, "Position Book")
+    except V20Error as e:
+        print(e)
+
+
+def create_order(accountID, instrument, other_args: List[str]):
+    parser = argparse.ArgumentParser(
+        add_help=False,
+        prog="create_order",
+        description="Create order",
+    )
+    parser.add_argument(
+        "-u",
+        "--unit",
+        dest="units",
+        action="store",
+        type=int,
+        default=0,
+        required=True,
+    )
+    parser.add_argument(
+        "-p",
+        "--price",
+        dest="price",
+        action="store",
+        type=float,
+        required=True,
+    )
+
+    ns_parser = parse_known_args_and_warn(parser, other_args)
+    if not ns_parser:
+        return
+
+    data = {
+        "order": {
+            "price": ns_parser.price,
+            "instrument": instrument,
+            "units": ns_parser.units,
+            "type": "LIMIT",
+            "timeInForce": "GTC",
+            "positionFill": "DEFAULT",
+        }
+    }
+    try:
+        request = orders.OrderCreate(accountID, data)
+        response = client.request(request)
+        print(response)
+    except V20Error as e:
+        print(e)
+
+
+def cancel_pending_order(accountID, other_args: List[str]):
+    parser = argparse.ArgumentParser(
+        add_help=False,
+        prog="cancelpendingorder",
+        description="Cancel Pending Order",
+    )
+    parser.add_argument(
+        "-i",
+        "--id",
+        dest="orderID",
+        action="store",
+        type=str,
+        required=True,
+    )
+    ns_parser = parse_known_args_and_warn(parser, other_args)
+    if not ns_parser:
+        return
+    try:
+        request = orders.OrderCancel(accountID, ns_parser.orderID)
+        client.request(request)
```

Comment:
Can you add a `print("")` at end.

Also, when it fails, it should fail just with a simple error message to the user.

<img width="959" alt="Captura de ecrã 2021-04-18, às 21 12 38" src="https://user-images.githubusercontent.com/25267873/115159400-d8e8d280-a08a-11eb-981b-ed6ac86f5e8c.png">